docs/index.rst

```diff
@@ -1426,35 +1432,47 @@ Setup |ccloud|
 
       ccloud login --save
 
-#. Use the :ref:`ccloud-stack` for a quick, automated way to create resources in |ccloud|.  Executed with a single command, it uses the |ccloud| CLI to:
+#. The remainder of the |ccloud| portion of this tutorial must be completed sequentially. We recommend that you manually complete all the steps in the following sections. However, you may also run the script :devx-cp-demo:`scripts/ccloud/create-ccloud-workflow.sh|scripts/ccloud/create-ccloud-workflow.sh` which automates those steps; this option is recommended for users who have run this tutorial before and want to quickly bring it up.
+
+   .. code-block:: text
+
+      (cd scripts/ccloud/ && ./create-ccloud-workflow.sh)
 
-   -  Create a new environment.
-   -  Create a new service account.
-   -  Create a new Kafka cluster and associated credentials.
-   -  Enable |sr-ccloud| and associated credentials.
-   -  Create ACLs with wildcard for the service account.
-   -  Generate a local configuration file with all above connection information.
+.. _cp-demo-ccloud-stack:
 
-   The first step is to get a bash library of useful functions for interacting with |ccloud| (one of which is ``cloud-stack``). This library is community-supported and is not supported by Confluent.
+ccloud-stack
+------------
+
+Use the :ref:`ccloud-stack` for a quick, automated way to create resources in |ccloud|.  Executed with a single command, it uses the |ccloud| CLI to:
+
+-  Create a new environment.
+-  Create a new service account.
+-  Create a new Kafka cluster and associated credentials.
+-  Enable |sr-ccloud| and associated credentials.
+-  Create ACLs with wildcard for the service account.
+-  Create a new ksqlDB app and associated credentials
+-  Generate a local configuration file with all above connection information.
+
+#. Get a bash library of useful functions for interacting with |ccloud| (one of which is ``cloud-stack``). This library is community-supported and is not supported by Confluent.
 
    .. code-block:: text
 
       curl -sS -o ccloud_library.sh https://raw.githubusercontent.com/confluentinc/examples/latest/utils/ccloud_library.sh
 
-#. Using ``ccloud_library.sh`` which you downloaded in the previous step, create a new ``ccloud-stack`` (see :ref:`ccloud-stack` for advanced options). It creates real resources in |ccloud| and takes a few minutes to complete.
+#. Using ``ccloud_library.sh`` which you downloaded in the previous step, create a new ``ccloud-stack`` (see :ref:`ccloud-stack` for advanced options). It creates real resources in |ccloud| and takes a few minutes to complete. The ``true`` flag adds creation of a ksqlDB application in |ccloud|, which has hourly charges even if you are not actively using it.
```

Comment:
should this be highlighted as a Caution? (charges)

Reply:
There are several other places in the docs where this is called out with more emphasis, so IMO don't think it needs to be repeated here.